src/redux/saga/employer.ts

```diff
@@ -1,19 +1,22 @@
-import { put, all, takeLatest, select } from 'redux-saga/effects';
-import axios from 'axios';
+import { put, takeLatest, select } from 'redux-saga/effects'
+import axios from 'axios'
+import { AppState } from '../types'
 
-import { registerEmployerSuccess } from '../../redux/actions/employer';
+import { registerEmployerSuccess } from '../../redux/actions/employer'
 
-function* registerEmployerSaga(credentials: Credential) {
+const credentials = (state: AppState) => state.employer.credentials
+function* registerEmployerSaga() {
+  const credentialData = yield select(credentials)
```

Comment:
I think you can put this inside the `try` block also for better format

Reply:
:+1: